src/discord-api.ts

```diff
@@ -114,6 +114,9 @@ export const getNewData = async (env: Env) => {
 	const updatedMessages: APIMessage[] = []
 	const deleteMessageIds: string[] = []
 
+	// To be deleted after attachments width and height are updated
+	const messagesWithAttachments: APIMessage[] = []
```

Comment:
I guess you could also just delete all attachments from the db and re-create them?

Reply:
I still need a code that will fetch all the messages and then look for the attachments in all of them so it would be the same.